### ISSUE TITLE
Remove 'Contracted outside of NZ' column?

### ISSUE THREAD

**Issue by m-kruth** [2kruman/COVID19-NZ-known-cases#29]
I feel that this cannot easily be understood from the data we are provided. For example the MOH provides data indicating that people used multiple modes of travel, from these it is not clear at which point the disease was contracted. I feel that since it is not clear and not a term explicitly used by MOH it should be removed from the dataset.

Instead  MOH is using `Community transmission` to refer to people who caught the virus and were not exposed to international travel or a person who travelled internationally. I feel that we should replace this column with this.

**Comment by RapidCompiler** [comment]
Should we have two columns named Community and International filled with boolean values (true or false) on the main data set?

**Comment by m-kruth** [comment]
Yes I think so, I've added more detail for what I think should be added to Issue #28
